test/Feature/RangeSpec.hs

```diff
@@ -79,11 +79,6 @@ spec = do
             simpleStatus r `shouldBe` ok200
 
       context "of invalid range" $ do
-        it "fails with 416 for offside range" $
-          request methodPost  "/rpc/getitemrange"
-                  (rangeHdrs $ ByteRangeFromTo 1 0) emptyRange
-            `shouldRespondWith` 416
-
```

Comment:
I think this test should stay and return an error for `ByteRangeFromTo 2 0`, for example. Another test could be made for Range 1-0 to return an empty array, because it could be interpreted as offset 1 for the element in position 0.